### DISCUSSION TITLE
choose computer

### DISCUSSION THREAD

**Discussion by 1phoenx** [community/community#141515] [Other Feature Feedback, Questions, & Ideas]
I am a sophomore college student in China, majoring in network engineering with a focus on cybersecurity. Which type of computer should I choose?

**Comment by VirajxD0** [comment]
Well mainly it depends on your budget.
but some options to consider:
Lenovo ThinkPad series
Dell XPS or Latitude series
HP Spectre or EliteBook series
Macbook Pro(if you prefer)
